builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -194,6 +194,10 @@ node {
 							}
 							events.setEnvironment(_envId)
 							events.sendStartedEvent('DELETE_ENVIRONMENT', "Environment cleanup for " + _envId + " started", null, _envId)
+              echo 'Service deletion is requested: ' + current_environment_id
+							if (is_service_deletion && service_config['event_source_s3']) { // check service for S3 Bucket and make empty if not. //
```

Comment:
what will be happen if an environment get deleted and the bucket is not empty?

Reply:
That is the actual issue, SLS unable to delete environment, when bucket is not empty.
i.e environment will not delete until bucket is empty in S3 case.

Reply:
Actually, my concern is,  We are manually deleting the content of the bucket, **only if (is_service_deletion && service_config['event_source_s3']) {** this condition is satisfying. Did you checked, this feature by deleting an environment ?

Reply:
Let us take this offline